### DISCUSSION TITLE
Zsh versions?

### DISCUSSION THREAD

**Discussion by Dmole** [ohmyzsh/ohmyzsh#10233] [Help] [answered]
```
zsh --version
```
Lists the installed version, but what is the version of the latest release (or the version of the master branch)?
Did I just miss it in the source and documentation?

**Comment by mcornella** [comment, marked as answer]
There's a GitHub mirror for `zsh` where you can look at [published releases](https://github.com/zsh-users/zsh/releases). The latest published one is 5.8.
